app/src/js/components/Granules/overview.js

```diff
@@ -63,24 +58,18 @@ class GranulesOverview extends React.Component {
     this.getExecuteOptions = this.getExecuteOptions.bind(this);
     this.applyRecoveryWorkflow = this.applyRecoveryWorkflow.bind(this);
     this.downloadGranuleCSV = this.downloadGranuleCSV.bind(this);
-    this.state = {};
+    this.state = {
+      workflow: this.props.workflowOptions[0]
+    };
   }
 
   componentDidMount () {
-    this.cancelInterval = interval(this.queryMeta, updateInterval, true);
-  }
-
-  componentWillUnmount () {
-    if (this.cancelInterval) { this.cancelInterval(); }
+    this.queryMeta();
   }
 
   queryMeta () {
     const { dispatch } = this.props;
     dispatch(listWorkflows());
-    dispatch(getCount({
-      type: 'granules',
-      field: 'status'
-    }));
```

Comment:
This getCount dispatch is taken care of in the index.js file. No need for it here

Reply:
This might be out of scope, but we did discuss making the overview cards values match what's in the displayed table.  I don't think this getCount dispatch works as written, but could it be updated to do that?  but there might already be a ticket for that.
![Screen Shot 2020-06-15 at 2 31 26 PM](https://user-images.githubusercontent.com/479480/84703518-9a880180-af15-11ea-94ff-d38d7dd9c265.png)

Reply:
> > This might be out of scope, but we did discuss making the overview cards values match what's in the displayed table. I don't think this getCount dispatch works as written, but could it be updated to do that? but there might already be a ticket for that.
> 
> @flamingbear Oh good call on this. I think it is out of scope, but is something to think about. Not seeing an existing ticket for it, so I made one.
> https://bugs.earthdata.nasa.gov/browse/CUMULUS-2029
> 
> Though I could see the argument that those cards are showing everything, regardless of what filters are applied, so might be worth discussing with the whole team

The ticket for this is CUMULUS-1868.